extensions/podman/src/extension.ts

```diff
@@ -853,15 +871,16 @@ export async function restartCurrentMachine(): Promise<void> {
 
   // Restart the current machine
   console.log('restarting autostarted machine', autoMachineName);
-  await execPromise(getPodmanCli(), ['machine', 'restart', autoMachineName]);
+  await extensionApi.process.exec(getPodmanCli(), ['machine', 'restart', autoMachineName]);
 }
 
 // Function that checks to see if the default machine is running and return a boolean
 export async function findRunningMachine(): Promise<string> {
   let runningMachine: string;
 
   // Find the machines
-  const machineListOutput = await execPromise(getPodmanCli(), ['machine', 'list', '--format', 'json']);
+  const machineListOutput = (await extensionApi.process.exec(getPodmanCli(), ['machine', 'list', '--format', 'json']))
+    .stdout;
```

Comment:
destructure

Also as it's like the 5 times that I see this command being called, maybe it needs its own function like

```
podmanMachineList(): string {
...
}
```

and we don't copy/paste `extensionApi.process.exec(getPodmanCli(), ['machine', 'list', '--format', 'json']` 5times in a row

Reply:
extracted call to the dedicate function: `getJSONMachineList`